azurerm/internal/services/web/tests/resource_arm_app_service_slot_test.go

```diff
@@ -3774,6 +3826,178 @@ resource "azurerm_app_service_slot" "test" {
 `, data.RandomInteger, data.Locations.Primary, data.RandomInteger, data.RandomInteger, data.RandomInteger)
 }
 
+func testAccAzureRMAppServiceSlot_detailedErrorMessagesEnabled(data acceptance.TestData) string {
```

Comment:
For these functions, could we add a bool to allow the function to set the value without needing to duplicate the full config, e.g.
```suggestion
func testAccAzureRMAppServiceSlot_detailedErrorMessagesEnabled(data acceptance.TestData, detailedEnabled bool) string {
```
This can then be passed in by referencing `%t`.